hubclient/component-client.go

```diff
@@ -86,3 +90,13 @@ func (c *Client) GetComponentVersion(link *hubapi.ResourceLink) (*hubapi.Compone
 func (c *Client) GetComponentVersionFromVariant(componentVariant *hubapi.ComponentVariant) (*hubapi.ComponentVersion, error) {
```

Comment:
I think this method should be removed, it does not follow the pattern of passing URL, which is whole library kind of follows.